ansible/roles/geoblacklight/tasks/geoblacklight_setup.yml

```diff
@@ -24,24 +24,29 @@
     group: "{{ project_group }}"
     mode: "0440"
 
-- name: install production gems
+- name: install the project's gems for development
   bundler:
     state: present
-    chdir: "{{ project_app_root }}"
-    exclude_groups: 'development test'
-    user_install: no
-  environment:
-    RAILS_ENV: '{{ project_app_env }}'
-  when: project_app_env == "production"
+    chdir: '{{ project_app_root }}'
+    binstub_directory: bin
+    gem_path: vendor/bundle
```

Comment:
I think we need `deployment_mode: False` here in case someone was using the production env and switches to development.